roles/bwc/tasks/bwc_repos_apt.yml

```diff
@@ -11,6 +11,9 @@
   retries: 5
   delay: 3
   until: _task is succeeded
+  tags:
+    - BWC repos
+    - StackStorm enterprise
```

Comment:
Please use the following tags:
```
tags:
  - ewc
  - enterprise
```
as well for other respective tasks where these tags were added